### PR TITLE
Kindly requesting a percentage type

### PR DESCRIPTION
**Pull request by joedevivo**
When using numbers to represent the percentage of some quantity, actually having a percentage type where you can use '%' is more intuitive and self-explanatory.  I imagine that always requiring the '%' makes sense.

### CODE REVIEW

**Comment by seancribbs** [comment]
:+1: 20040a3

File: src/cuttlefish_datatypes.erl [hunk 1]
```diff
@@ -209,6 +238,32 @@ from_string(Flag, flag) when is_atom(Flag) -> cuttlefish_flag:parse(Flag);
 from_string(Flag, {flag, _, _}=Type) when is_list(Flag) -> cuttlefish_flag:parse(Flag, Type);
 from_string(Flag, {flag, _, _}=Type) when is_atom(Flag) -> cuttlefish_flag:parse(Flag, Type);
 
+from_string(Percent, {percent, integer}) when is_integer(Percent), Percent >= 0, Percent =< 100 -> Percent;
+from_string(Percent, {percent, integer}) when is_integer(Percent) ->
+    {error, lists:flatten(io_lib:format("~p% can't be outside the range 0 - 100%", [Percent]))};
```
Comment (engelsanchez) [inline src/cuttlefish_datatypes.erl:243]: Should have added "What are you, a basketball coach?" here.

Comment (joedevivo) [inline src/cuttlefish_datatypes.erl:243]: lol, you are right about that

**Comment by joedevivo** [comment]
@borshop merge

**Comment by joedevivo** [comment]
this also resolves #149

**Pull request merged**